hedera-node/src/main/java/com/hedera/services/exceptions/ValidationUtils.java

```diff
@@ -49,6 +54,15 @@ public static void validateFalse(final boolean flag, final ResponseCodeEnum code
 		}
 	}
 
+	//TODO: Ask for Yoan's and Dani's approval for this new method. If approved add JavaDoc
+	public static void validateFalseWithLogging(final boolean flag, final ResponseCodeEnum code, final TopicID topicID) {
+		if (flag) {
+			log.error("Update topic {} has invalid admin key specified, " +
+					"which should have been caught during signature validation", topicID);
+			throw new InvalidTransactionException(code);
+		}
+	}
+
```

Comment:
If you are intending to create the method with logging as the only additional functionality, different from `validateFalse`, I think you can do two things:
1) make it better by generalizing the logging message as a parameter
2) remove it entirely since it is essentially the same method as `validateFalse`

Reply:
I think that option 2 is actually the better one. Imo it is safe to remove the logger and use the original `validateFalse` method